### DISCUSSION TITLE
Markdown heading button

### DISCUSSION THREAD

**Discussion by digitall-it** [filamentphp/filament#4601] [Help] [answered]
As the markdown editor can interpret markdown headings (i.e. H1, H2, H3), I was pretty sure I could enable it with toolbarButtons, but alas, either I miss the syntax, or it isn't implemented. What's the shortest way to add this button?

**Comment by zepfietje** [comment, marked as answer]
The Markdown editor currently doesn't have toolbar buttons for headings.
